streaming/src/main/scala/org/apache/spark/streaming/receiver/ReceiverSupervisor.scala

```diff
@@ -23,9 +23,9 @@ import scala.collection.mutable.ArrayBuffer
 
 import org.apache.spark.{Logging, SparkConf}
 import org.apache.spark.storage.StreamBlockId
-import java.util.concurrent.CountDownLatch
 import scala.concurrent._
 import ExecutionContext.Implicits.global
+import java.util.concurrent.CountDownLatch
```

Comment:
We don't actually have to modify this file as part of this patch, though, so it'd be fine to just roll it back to what's in master.